src/test/java/pageObjects/TagPageTests.java

```diff
@@ -0,0 +1,50 @@
+package pageObjects;
+
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Objects;
+import java.util.Random;
+
+public class TagPageTests extends BaseTest {
+
+    private String addNewTag(String newTagName) {
+        if (Objects.equals(newTagName, "")) {
+            Random rand = new Random();
+            newTagName = "tag-test-" + rand.nextInt();
+        }
+
+        tagPage.AddTag(newTagName);
+        return newTagName;
+    }
```

Comment:
this function should be placed in another package (for example utils)

Reply:
This is strictly for tag tests not a generic helper function

Reply:
You should add it in main-> java -> pageObject package, as it's a function not a test.